Source/WebCore/css/parser/CSSSupportsParser.cpp

```diff
@@ -147,13 +147,32 @@ CSSSupportsParser::SupportsResult CSSSupportsParser::consumeSupportsSelectorFunc
     return CSSSelectorParser::supportsComplexSelector(block, m_parser.context(), m_isNestedContext) ? Supported : Unsupported;
 }
 
+
+CSSSupportsParser::SupportsResult CSSSupportsParser::consumeSupportsFontFormatFunction(CSSParserTokenRange& range)
+{
+    if (range.peek().type() != FunctionToken || range.peek().functionId() != CSSValueFontFormat)
+        return Invalid;
```

Comment:
You make this same check before entering the method. If you worry you might forget to check before entering this code, maybe it should just be an ASSERT?